vision/cloud-client/document_text/env/.Python

```diff
@@ -0,0 +1 @@
+/System/Library/Frameworks/Python.framework/Versions/2.7/Python
```

Comment:
Did you mean to include these env files?

Reply:
That file is outdated and creds are revoked.